Alerting: Add support for TTL for pushover for Mimir Alertmanager (#78687)
grafana/grafana: docs/sources/old-alerting/notifications.md

| OK priority    | The priority OK notifications are sent; if not set, then OK notifications are sent with the priority set for alerting notifications |
| Retry          | How often (in seconds) the Pushover servers send the same notification to the user. (minimum 30 seconds)                            |
| Expire         | How many seconds your notification will continue to be retried for (maximum 86400 seconds)                                          |

> | TTL            | A number of seconds that the message will live, before being deleted automatically                                                  |
> | Alerting sound | The sound for alerting notifications                                                                                                |

Reviewer: ```suggestion
| TTL            | The number of seconds before a message expires and is deleted automatically                                                  |
| Alerting sound | The sound for alerting notifications                                                                                                |
```

> | Alerting sound | The sound for alerting notifications                                                                                                |

Reviewer: Change this message to be the same as the one below - the number of seconds...
